### PR TITLE
Publish 1.7.1

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Use new @ionic/cli tool

### CODE REVIEW

**Comment by huayunh** [comment]
I was able to generate an ionic project using the blank and the basic routing templates. Re-ran CI but the build_ionic is still failing.

**Comment by huayunh** [comment]
I also noticed that a `package-lock.json` instead of `yarn.lock` was generated. Is this the intended behavior of the ionic cli?

**Comment by emclaug2** [comment]
Yeah this is a little strange; I'll look into the pipeline to figure out why it's failing.  But @huayunh yes unfortunately the @ionic/cli doesn't allow us to choose between npm or yarn, we're forced to install via npm so ionic projects come with a package-lock by default.

**Comment by joebochill** [comment]
> Yeah this is a little strange; I'll look into the pipeline to figure out why it's failing. But @huayunh yes unfortunately the @ionic/cli doesn't allow us to choose between npm or yarn, we're forced to install via npm so ionic projects come with a package-lock by default.

Angular is the same...it only uses npm, not yarn too.

File: src/extensions/project-extensions.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ module.exports = (toolbox: GluegunToolbox): void => {
             QUESTIONS.template,
         ]);
 
-        const command = `${NPM7_PREFIX} && npx ionic start ${name} blank`;
+        const command = `${NPM7_PREFIX} && npx @ionic/cli start ${name} blank`;
```
Comment (joebochill) [inline src/extensions/project-extensions.ts:109]: When the name of the package and the command are different, you need to use the -p syntax:

```suggestion
        const command = `${NPM7_PREFIX}&& npx -p @ionic/cli ionic start ${name} blank`;
```

**Comment by jeffvg** [comment]
verified all 3 frameworks, all 3 templates and ionic ios/android

**Review by jeffvg — APPROVED** [review]
verified all 3 frameworks, all 3 templates and ionic ios/android

**Comment by joebochill** [comment]
> verified all 3 frameworks, all 3 templates and ionic ios/android

Damn Jeff, you're on the ball :P

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.7.1 (May 13, 2021)
```
Comment (joebochill) [inline CHANGELOG.md:3]: Unfortunately, it's now May 14 :(

**Review by joebochill — APPROVED** [review]

**Pull request merged**